base/BUILD

```diff
@@ -60,7 +60,15 @@ intellij_plugin_library(
     plugin_xmls = [
         "src/META-INF/dependency-on-lvcs-impl.xml",
         "src/META-INF/blaze-base.xml",
-    ],
+    ] + select_for_plugin_api({
+        "intellij-2024.3": ["src/META-INF/blaze-base-243.xml"],
+        "intellij-2025.1": ["src/META-INF/blaze-base-251.xml"],
+        "intellij-ue-2024.3": ["src/META-INF/blaze-base-243.xml"],
+        "intellij-ue-2025.1": ["src/META-INF/blaze-base-251.xml"],
+        "clion-2024.3": ["src/META-INF/blaze-base-243.xml"],
+        "clion-2025.1": ["src/META-INF/blaze-base-251.xml"],
+        "default": [],
```

Comment:
This looks like maintenance burden to me. Can we solve this similar to `//clwb/src/com/google/idea/blaze/clwb/radler` and use something like `select_since` instead of introducing all these xml files.